dll/win32/kernel32/kernel32_vista/EnumSystemLocalesEx.c

```diff
@@ -0,0 +1,51 @@
+#include "k32_vista.h"
+#include <winuser.h>
+
+struct enum_locale_ex_data
+{
+    LOCALE_ENUMPROCEX proc;
+    DWORD             flags;
+    LPARAM            lparam;
+};
+
+static BOOL CALLBACK enum_locale_ex_proc( HMODULE module, LPCWSTR type,
+                                          LPCWSTR name, WORD lang, LONG_PTR lparam )
+{
+    struct enum_locale_ex_data *data = (struct enum_locale_ex_data *)lparam;
+    WCHAR buffer[256];
+    DWORD neutral;
+    unsigned int flags;
+
+    GetLocaleInfoW( MAKELCID( lang, SORT_DEFAULT ), LOCALE_SNAME | LOCALE_NOUSEROVERRIDE,
+                    buffer, sizeof(buffer) / sizeof(WCHAR) );
```

Comment:
nit: `countof` or one of the 300 variations of it?

Reply:
It's wine code and will eventually be replaced with wine's kernelbase.